site/src/utilities.ts

```diff
@@ -259,7 +259,9 @@ function getPrerequisiteFormatHtml(
       .map((childPrereq) =>
         getPrerequisiteFormatHtml(priorCourses, childPrereq, false)
       )
-      .join(` ${prereq.type} `);
+      .join(
+              prereq.type == "and" ? " and "+(topLevel ? "<br />" : "") : " or "
+      );
```

Comment:
I'm not a fan of breaking the prereqs into multiple lines on `and` requirements (that looks like what this logic does).  It won't render cleanly when the info card is too narrow and since there isn't significant spacing between lines there's not much of a readability increase from this change.

Reply:
That is what the logic does; I did this because I found it easier to read when I was testing, and just left it in. This change can also be dropped, and it doesn't have much useful effect anyway.

Reply:
Alright, in that case let's leave it as the non-breaking version.